src/controllers/locationSearch.controller.js

```diff
@@ -0,0 +1,22 @@
+import sequelize from 'sequelize';
+import database from '../database/models';
+import { successResponse, errorResponse, serverError } from '../helpers/response';
+
+const getByLocation = async (req, res) => {
+    try { 
+        const vendors = await database.Vendor.findAll({
+            where: {
+                location: req.query.location,
```

Comment:
This might not be optimal, the `location` value might not be strictly equal to `query.location`, can't we use something like `location like` or `location includes`. Research more on it please.

Reply:
Whatever location the user inputs which is the query location, if it matches any location stored in the database then the user gets the results if not a message pops up. However, other options relating to the query location could come up as alternative options that's where we can add the other attributes ( like or include) if possible.